docs/contributing/test-data-models-overview.md

```diff
@@ -0,0 +1,250 @@
+- `presets`: predefined versions of the data model
+
+This is how it looks like the entry point of a data model package:
+
+```ts
+import { RestModelBuilder, GraphqlModelBuilder } from './builders';
+import * as GeometryPresets from './presets';
+
+export * from './types';
+
+export const GeometryRest = {
+  random: RestModelBuilder,
+  presets: GeometryPresets.restPresets,
+};
+
+export const GeometryGraphql = {
+  random: GraphqlModelBuilder,
+  presets: GeometryPresets.graphqlPresets,
+};
+```
+
+### Examples
+
+#### Building default random data models
+
+```ts
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  type TGeometryRest,
+  type TGeometryGraphql,
+} from '@commercetools-test-data/geometry';
+
+const restGeometry: TGeometryRest = RestModelBuilder.random().build();
+const graphqlGeometry: TGeometryGraphql = GraphqlModelBuilder.random().build();
+```
+
+#### Building customized data models
+
+```ts
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  type TGeometryRest,
+  type TGeometryGraphql,
+} from '@commercetools-test-data/geometry';
+
+const restGeometry: TGeometryRest = RestModelBuilder.random()
+  .coordinates([12.45, 66.33])
+  .build();
+const graphqlGeometry: TGeometryGraphql = GraphqlModelBuilder.random()
+  .coordinates([12.45, 66.33])
+  .build();
+```
+
+#### Building predefined version data models
+
+```ts
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  type TGeometryRest,
+  type TGeometryGraphql,
+} from '@commercetools-test-data/geometry';
+
+const restGeometry: TGeometryRest = RestModelBuilder.presets.munich().build();
+const graphqlGeometry: TGeometryGraphql = GraphqlModelBuilder.presets
+  .munich()
+  .build();
+```
+
+## Internal structure
+
+The way a data model is implemented is by using some classes and helpers exposed from the [core](https://github.com/commercetools/test-data/tree/main/core) package.
+
+### Fields config
+
+The core file is named `fields-config` and its responsibility is to describe how the data model properties should be pre-populated and also resolve dependencies among them (if any).
+
+Here's an example of a simple one:
+
+```ts
+import { fake, type TModelFieldsConfig } from '@commercetools-test-data/core';
+import type { TGeometryGraphql, TGeometryRest } from './types';
+
+const commonFieldsConfig = {
+  type: fake(() => 'Point'),
+  coordinates: fake((f) => [f.location.longitude(), f.location.latitude()]),
+};
+
+export const restFieldsConfig: TModelFieldsConfig<TGeometryRest> = {
+  fields: {
+    ...commonFieldsConfig,
+  },
+};
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TGeometryGraphql> = {
+  fields: {
+    ...commonFieldsConfig,
+    __typename: 'Geometry',
+  },
+};
+```
+
+As you can see, this files is required to export two configuration objects (`restFieldsConfig`, `graphqlFieldsConfig`) which describe how to populate the properties of the data model when building the default version.
+
+The configuration objects has two properties:
```

Comment:
```suggestion
The configuration object has two properties:
```

Reply:
I used plural on purpose because we have two configuration objects, one for rest and another for graphql, and I'm trying to refer to them.

Reply:
```suggestion
The configuration objects have two properties:
```

Reply:
I think this is closer to your intended meaning here?

Reply:
Thanks Tyler 🙇 

Updated here: 7b0f7aaddbe19f5d8fd2a8af21a21edc0c90a3c9